README.md

```diff
@@ -9,6 +9,7 @@ Allows to periodically collect measurements and dispatch them as Telemetry event
 
   * `[vm, memory]` - contains the total memory, process memory, and all other keys in `erlang:memory/0`
   * `[vm, total_run_queue_lengths]` - returns the run queue lengths for CPU and IO schedulers. It contains the `total`, `cpu` and `io` measurements
+  * `[vm, system_limits]` - returns the current process, atom and port count as per `erlang:system_info/1`
```

Comment:
Did you think of an alternative name? `limits` suggests that it's a constant that can't be exceeded, but it's not what this event includes.

How about `system_resources`? Although it might be too vague..

Reply:
`system_counts` maybe? Then the measurements are `system_counts.atom`, `system_counts.process`, `system_counts.port`?

Reply:
Should I change it to `system_counts` then?

Reply:
@arkgil @bryannaegele what is your take?

Reply:
Yup, system counts sounds good to me 👍